tools/races.js

```diff
@@ -32,14 +33,14 @@ const defaultRaces: Record<string, Record<Race, number>> = ${JSON.stringify(
 )};
 
 if (isSport("hockey")) {
-	defaultRaces.USA = {
-		asian: 2,
-		black: 2,
-		brown: 3,
-		white: 93,
-	};
-	defaultRaces.Germany = defaultRaces.USA;
-	defaultRaces.Spain = defaultRaces.USA;
+    defaultRaces.USA = {
+        asian: 2,
+        black: 2,
+        brown: 3,
+        white: 93,
+    };
+    defaultRaces.Germany = defaultRaces.USA;
+    defaultRaces.Spain = defaultRaces.USA;
```

Comment:
Not sure how this got changed, but prettier is supposed to be running on every commit, which would change this back to tabs. It is not a huge issue, but if you're going to be doing other PRs, it would be good to figure out if prettier is not working correctly, I guess it could be another Windows issue :)

Reply:
I checked the outputs for Prettier and thankfully it is linting properly, this was actually just a silly extension in my IDE setup. I've disabled it now and restored the formatting for this section